test.js

```diff
@@ -4,15 +4,15 @@ const duckdb_native = require('.');
 console.log("DuckDB version:", duckdb_native.duckdb_library_version());
 
 function convert_validity(vector, n) {
-    const res = new Uint8Array(n).fill(true);
+    const res = Array.from({ length: n }).fill(true);
```

Comment:
It seems like this function should return a normal JS array of booleans. This is a more canonical way to do that.